src/nvim/screen.c

```diff
@@ -6400,7 +6400,7 @@ void screenalloc(void)
   if ((default_grid.chars != NULL
        && Rows == default_grid.Rows
        && Columns == default_grid.Columns
-       )
+  )
```

Comment:
this just doesn't make sense at all (there's nothing at this indent level to refer back to)

Reply:
it's aligned with the "if". it's a fairly common style (but i have no preference).

Reply:
> it's aligned with the "if". it's a fairly common style (but i have no preference).

except that it is _not_ the `)` that belongs to the `if(` statement but a _inner expression_ parenthesis. Look at the full diff, not just what github's UI shows you.

Reply:
This is a weird quirk of uncrustify I'm still trying to get the hang of. It can sometimes struggle with code that is "supposed to be" in a single line, but isn't. This was the exact same scenario in keymap.h with the weird indentation if you remember. I'm looking through the options but the worst case scenario is that we just fix it manually (join it in one line) and uncrustify will take care of it properly afterwards.

Reply:
The minimal change _for uncrustify_ would be for it to just move the end bracket to the previous line (and not automatically join lines which is often correct but not always)

Reply:
fixed